docs/runbooks/VirtualMachineStuckInUnhealthyState.md

```diff
@@ -0,0 +1,209 @@
+# VirtualMachineStuckInUnhealthyState
+
+## Meaning
+
+This alert fires when a VirtualMachine has been stuck in an unhealthy state for more than 10 minutes and does not have an associated VMI (VirtualMachineInstance).
+
+The alert indicates that a VirtualMachine is experiencing early-stage lifecycle issues before a VMI can be successfully created.
+This typically occurs during the initial phases of VM startup when KubeVirt is trying to provision resources, pull images, or schedule the workload.
+
+**Affected States:**
+- `provisioning` - Resources (DataVolumes, PVCs) are being prepared
+- `starting` - VM is attempting to start but no VMI exists yet
+- `terminating` - VM is being deleted but without an active VMI
+- `Error` states - Various scheduling, image, or resource allocation errors
+
+## Impact
+
+- **Severity:** Warning
+- **User Impact:** VMs cannot start or are stuck in error states
+- **Business Impact:** Workloads cannot be deployed, affecting application availability
+
+## Possible Causes
+
+### Resource-Related Issues
+- **Insufficient cluster resources** (CPU, memory, storage)
+- **Missing or misconfigured storage classes**
+- **PVC provisioning failures**
+- **DataVolume creation/import failures**
+
+### Image and Registry Issues  
+- **Container image pull failures** for containerDisk volumes
+- **Registry authentication problems**
+- **Network connectivity issues to image registries**
+- **Missing or corrupted VM disk images**
+
+### Scheduling and Node Issues
+- **No schedulable nodes available** (all nodes cordoned/unschedulable)
+- **Insufficient KVM resources** on available nodes
```

Comment:
Can also be other resources, e.g. GPU or a mismatch between requested and available CPU models

Reply:
Updated